.github/workflows/render-and-upload-manifests-reusable.yml

```diff
@@ -35,6 +35,14 @@ jobs:
         # Note: your repository needs to have this script.
         run: ./hack/scripts/render_crd.sh "${VERSION}"
 
+      - name: Rename CR file
+        shell: bash
+        env:
+          CR_FILE: ${{ inputs.CR_FILE }}
+        run: |
+          # this is where all our default CRs are located.
+          ln -s config/samples/default.yaml "./${CR_FILE}"
```

Comment:
The name says `Rename CR file` but the step actually creates symbolic link!

Reply:
I thought "Create a symlink" might be harder to understand and the purpose of it might be even harder to understand. On the other hand, creating a symlink is a much cheaper operation compared to `cp` or `mv` but has a similar result.

Reply:
So, should I rename the step, or should I change the operation to `cp` or `mv`?

Reply:
IMO, rename it to `Create Symlink for CR`.
Also if the symlink already exists, this command will fail, otherwise you can use `ln -sf`.